addons/mail/static/src/xml/wip/widget/systray_messaging_menu_dropdown.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+
+    <div owl="1" t-name="mail.wip.widget.SystrayMessagingMenuDropdown" class="o_mail_systray_dropdown dropdown-menu dropdown-menu-right" role="menu">
+        <div t-if="props.mobile" class="o_mail_systray_mobile_header">
+            <button type="button" class="btn btn-secondary o_new_message"> New message </button>
+        </div>
+        <div t-else="1" class="o_mail_systray_dropdown_top">
+            <div>
+                <button t-on-click='_onClickFilter' type="button" t-attf-class="btn btn-link o_filter_button {{!state.filter ? 'active' : ''}}" role="tab"> All </button>
+                <button t-on-click='_onClickFilter' type="button" t-attf-class="btn btn-link o_filter_button {{state.filter === 'chat' ? 'active' : ''}}" data-filter='chat' role="tab"> Chat </button>
+                <button t-on-click='_onClickFilter' type="button" t-attf-class="btn btn-link o_filter_button {{state.filter === 'channels' ? 'active' : ''}}" data-filter='channels' role="tab"> Channels </button>
+            </div>
+            <button type="button" class="btn btn-link o_new_message"> New message </button>
+        </div>
+        <t t-widget="ThreadPreviewList" class="o_mail_systray_dropdown_items" t-props="{ filter: state.filter }"/>
```

Comment:
should not re-render it each time a message arrives if the dropdown is not opened?